fixieai/agents/corpora.py

```diff
@@ -1,18 +1,147 @@
+import base64
+import dataclasses
 from typing import List, Optional
 
+import dataclasses_json.core
 from pydantic import dataclasses as pydantic_dataclasses
 
 
 @pydantic_dataclasses.dataclass
-class DocumentLoader:
-    """Document loader for a Fixie CodeShot Agent."""
+class CorpusRequest(dataclasses_json.DataClassJsonMixin):
```

Comment:
Removing `DocumentLoader` seems like a breaking change? Would recommend leaving it for now unless we need to remove it.

Reply:
AFAICT, this never did anything so I figured we could bump the version to be safe and remove it.  For now I've added it back and deprecated it instead.